spark2/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -298,4 +309,69 @@ public void testImportWithNameMappingForVectorizedParquetReader() throws Excepti
 
     Assert.assertEquals(expected.stream().map(SimpleRecord::getData).collect(Collectors.toList()), actual);
   }
+
+  @Test
+  public void testAvroReaderWithNameMapping() throws IOException {
```

Comment:
Why was this added to `SparkTableUtil`? It doesn't use `SparkTableUtil`, so it is not related.

Could you start a test suite, `TestNameMappingProjection`?

Reply:
will do.